PWGCF/FemtoUniverse/Tasks/femtoUniversePairTaskTrackTrackMultKtExtended.cxx

```diff
@@ -254,7 +254,6 @@ struct femtoUniversePairTaskTrackTrackMultKtExtended {
         return false;
       }
     }
-    return false;
   }
 
   bool IsKaonNSigma(float mom, float nsigmaTPCK, float nsigmaTOFK)
```

Comment:
This is a copy of the function IsProtonSigma(). Couldn't all the functions Is...Sigma() be merged into one?
This way, in the future you will need to modify the code just in 1 place instead of 4.

Reply:
Yes, but those functions are later used separately in a 'switch-case' construction in IsParticleNSigma() function depending on PDG codes, I should not think that merging them all into one would be a successful idea.

Reply:
I still don't see a problem.
In IsParticleNSigma(), you would have lines like `return IsNSigma(mom, nsigmaTPCPr, nsigmaTOFPr);`, `IsNSigma(mom, nsigmaTPCK, nsigmaTOFK);` instead of `return IsProtonNSigma(mom, nsigmaTPCPr, nsigmaTOFPr);`, `IsKaonNSigma(mom, nsigmaTPCK, nsigmaTOFK);`.
It is just a change in the name of the function. The rest works the same way.